src/lib.rs

```diff
@@ -590,6 +592,52 @@ impl<T: Debug + PrimInt + One + Zero> Vob<T> {
         }
     }
 
+    /// Append all elements from a second Vob to this Vob.
+    ///
+    /// # Examples
+    /// ```
+    /// #[macro_use] extern crate vob;
+    /// fn main() {
+    ///     let mut v1 = vob![true];
+    ///     let v2 = vob![false, false];
+    ///     v1.extend_from_vob(&v2);
+    ///     assert_eq!(v1, vob![true, false, false]);
+    /// }
+    /// ```    
+    pub fn extend_from_vob(&mut self, other: &Vob<T>) {
+        self.reserve(other.len());
+        let ub = self.len % bits_per_block::<T>();
+        // If there are no unused bits, we can just push the new blocks
```

Comment:
I think this comment should be inside the `if` (to match the comment in the `else` branch).